index.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html>
+    <head>
+
```

Comment:
You should add a title tag. That will give the tab a name in the browser.

Reply:
Done